### DISCUSSION TITLE
No sound in clips?

### DISCUSSION THREAD

**Discussion by flaektrem** [blakeblackshear/frigate#590] [Ask A Question] [answered]
Is it possible to get sound in clips?
RTMP stream of my cameras have sound, but not recorded clips.

Thanks

**Comment by blakeblackshear** [comment, marked as answer]
Override and remove the `-an` from the output args for record and clips: https://blakeblackshear.github.io/frigate/configuration/index#ffmpeg

**Reply by flaektrem** [reply]
Ahh, that was easy, thank you!

Maybe add som notes about that in the docs?
